### PR TITLE
Fixes Issue/#20 and Issue/#39

### LINKED ISSUE #20: running adapt install in a non-adapt folder yields no feedback
it should say "hey this is not an adapt folder" or something

### LINKED ISSUE #39: Warn user on malformed adapt.json
Process just dies quietly if this happens (i.e. a trailing comma), leading the user to believe everything's gone smoothly.

### PR DESCRIPTION
**Pull request by hrajotia**
- Fixes #20, Now showing below error message for command like `install`, `uninstall`, and `ls` :
  `Fatal error: please run above commands in adapt course directory.`
- Fixes #39, Malformed JSON errors now reported using [json-lint](https://www.npmjs.com/package/json-lint).

### CODE REVIEW

**Comment by oliverfoster** [comment]
+1

**Comment by moloko** [comment]
Reading through the code, I think this looks OK - but it feels like the sort of thing I should test really! Is that something that's relatively easy to do? Or have you already done so @oliverfoster ?

**Comment by hrajotia** [comment]
@moloko: I follow below steps to develop and test features for adapt-cli locally -
* Clone adapt-cli repo and switch to respective branch (e.g. `few_issues`)
* Do `npm install` there
* Creating a `index.js` file on same directory and paste below code there
  ```javascript
    var cli = require('./lib/cli');
    cli.withOptions().withPackage().execute();
  ```

* Now using cmd execute commands (like: `create`, `search`, `ls`, etc) with following syntax
  ```shell
    $ node index create course
    $ node index search <some_text>
  ```

File: package.json [hunk 1]
```diff
@@ -13,6 +13,7 @@
     "debug": "~2.1.3",
     "download": "git://github.com/adaptlearning/download.git",
```
Comment (chris-steele) [inline package.json]: Just need to update this to "download": "~5.0.2"

**Comment by chris-steele** [comment]
Despite 18 months elapsing, only one minor conflict. Apologies for the length of time taken to get this reviewed @hrajotia

**Comment by hrajotia** [comment]
Hey there, its was pleasure working for this amazing project.. Past few months I got busy in other activities around me so didn't get chance to look upon this outstanding PR.

@chris-steele thanks for fixing the conflict, By looking at Travis log, its seems to me that I somehow forgot to add json-lint to package.json. can you please add that too, as currently I do not have write access to this repo... thanks and cheers guys..

**Comment by hrajotia** [comment]
@chris-steele thanks!

**Comment by chris-steele** [comment]
@hrajotia No problem and thanks for noticing this - I should have noticed this myself!

**Pull request merged**